UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneImporter/ImporterTextures.cs

```diff
@@ -202,9 +206,22 @@ protected virtual async Task ConstructUnityTexture(Stream stream, bool markGpuOn
 
 		protected virtual int GetTextureSourceId(GLTFTexture texture)
 		{
+			if (texture.Extensions != null && texture.Extensions.ContainsKey(KHR_texture_basisu.EXTENSION_NAME))
+			{
+				return (texture.Extensions[KHR_texture_basisu.EXTENSION_NAME] as KHR_texture_basisu).source.Id;
```

Comment:
Should we guard against empty "source"?

Reply:
825268c674f3de83335c782103f09cc3f77678f1